Cargo.toml

```diff
@@ -61,7 +61,7 @@ noirc_frontend = { path = "compiler/noirc_frontend" }
 noirc_printable_type = { path = "compiler/noirc_printable_type" }
 noir_wasm = { path = "compiler/wasm" }
 cfg-if = "1.0.0"
-clap = { version = "4.3.19", features = ["derive"] }
+clap = { version = "=4.3.19", features = ["derive"] }
```

Comment:
This seems to be getting changed by a transitive dependency to a higher version, but higher versions are not compatible with rust 1.66

Reply:
The equals makes it clear that we definitely want 4.3.19

Reply:
Removed and now only adding MPL2.0